src/VisualStudio/Core/Test/CodeModel/AbstractRootCodeModelTests.vb

```diff
@@ -39,9 +39,8 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.CodeModel
         Protected Overloads Sub TestChildren(code As XElement, ParamArray names() As String)
             TestRootCodeModelWithCodeFile(code,
                 Sub(rootCodeModel)
-                    Assert.Equal(names.Length, rootCodeModel.CodeElements.Count)
-
                     Dim actualNames = rootCodeModel.CodeElements.OfType(Of EnvDTE.CodeElement).Select(Function(e) e.Name).ToArray()
+                    Assert.Equal(names.Length, rootCodeModel.CodeElements.Count)
```

Comment:
unintentional change?

Reply:
This made debugging failures easier (you get to access the `actualNames` local)